dev/WindowsAppRuntime_BootstrapDLL/WindowsAppRuntime.Bootstrap.rc

```diff
@@ -0,0 +1,33 @@
+#include <winresrc.h>
+
+#define STR1(x) #x
+#define DELIM STR1(.)
+#define STR2(a,b)     STR1(a) DELIM STR1(b)
+
+#define WAS_RELEASE_MAJOR           1
+#define WAS_RELEASE_MINOR           2
+#define VER_PRODUCT_VERSION         WAS_RELEASE_MAJOR, WAS_RELEASE_MINOR, 0, 0
+#define VER_PRODUCT_VERSION_STRING  STR2(WAS_RELEASE_MAJOR, WAS_RELEASE_MINOR)
+
+VS_VERSION_INFO VERSIONINFO
+ PRODUCTVERSION VER_PRODUCT_VERSION
+ FILEFLAGSMASK 0x3fL
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "Microsoft Corporation"
+            VALUE "FileDescription", "WindowsAppRuntime.Bootstrap.dll"
+            VALUE "InternalName", "WindowsAppRuntime.Bootstrap.dll"
+            VALUE "LegalCopyright", "Copyright (c) Microsoft Corporation. All rights reserved."
+            VALUE "OriginalFilename", "WindowsAppRuntime.Bootstrap.dll"
```

Comment:
`Microsoft.WindowsAppRuntime.Bootstrap.dll`

Reply:
You can't look at the project filename for the output filename. Not for .vcxproj

Look in the .vcxproj at the `TargetName` element e.g. `<TargetName>Microsoft.WindowsAppRuntime.Bootstrap.dll</TargetName>`